Gemfile.lock

```diff
@@ -262,83 +294,85 @@ GEM
       rb-inotify (~> 0.9, >= 0.9.7)
       ruby_dep (~> 1.2)
     logger (1.2.8)
-    loofah (2.0.3)
+    loofah (2.2.1)
+      crass (~> 1.0.2)
       nokogiri (>= 1.5.9)
-    mail (2.6.6)
-      mime-types (>= 1.16, < 4)
-    marc (1.0.0)
+    mail (2.7.0)
+      mini_mime (>= 0.1.1)
+    marc (1.0.2)
       scrub_rb (>= 1.0.1, < 2)
       unf
     marc-fastxmlwriter (1.0.0)
       marc (~> 1.0)
-    method_source (0.8.2)
-    mime-types (3.1)
-      mime-types-data (~> 3.2015)
-    mime-types-data (3.2016.0521)
+    method_source (0.9.0)
+    mini_mime (1.0.0)
     mini_portile2 (2.3.0)
-    minitest (5.10.3)
-    mods (2.1.0)
+    minitest (5.11.3)
+    mods (2.3.1)
```

Comment:
Mostly the date parsing stuff we did for exhibits: https://github.com/sul-dlss/mods/compare/v2.1.0...HEAD